src/command/restore/restore.c

```diff
@@ -2073,6 +2074,34 @@ restoreProcessQueue(const Manifest *const manifest, List **const queueList)
             const ManifestFilePack *const filePack = manifestFilePackGet(manifest, fileIdx);
             const ManifestFile file = manifestFileUnpack(manifest, filePack);
 
+            Oid dbNode, relNode;
+            // If this file is located in the default tablespace
+            if (sscanf(strZ(file.name), MANIFEST_TARGET_PGDATA "/" PG_PATH_BASE "/%u/%u", &dbNode, &relNode) == 2)
```

Comment:
How much does the sscanf call slow down the execution? And why do we need it if the filter is not set at all?